.circleci/config.yml

```diff
@@ -1,6 +1,19 @@
-version: 2.0
+version: 2.1
 
 jobs:
+  pre-commit:
+    docker:
+      - image: ubuntu:20.10
```

Comment:
Question: Why not an LTS?

Reply:
because `pre-commit` package is being included since 20.10